math/src/test/kotlin/ktx/math/rangesTest.kt

```diff
@@ -232,4 +235,18 @@ class RangesTest {
     }
     assert(values.all { it in range })
   }
+
+  @Test
+  fun `should interpolate`() {
+    val start = Random.nextFloat()
+    val end = Random.nextFloat()
+    val progress = Random.nextFloat()
```

Comment:
Avoid using random values in tests, unless the seed is set beforehand.

In this case I'd use fixed values.